docs/getting-started/usage/navigating-the-main-menu.en.md

```diff
@@ -58,15 +58,15 @@ Words are converted to their BIP-39 numeric indexes, those numbers are then conc
 <img src="../../../img/maixpy_m5stickv/backup-stackbit-125.png" align="right">
 <img src="../../../img/maixpy_amigo/backup-stackbit-150.png" align="right">
 
-Words are converted to their BIP-39 numeric indexes, then each of the four digits is converted to a sum of 1,2,4 or 8.
+Words are converted to their BIP-39 numeric indexes, then each of the four digits is converted to a sum of 1,2,4 or 8. This option does not print even if a printer driver is set.
```

Comment:
I believe that "numeric indexes" might be more clear as "word numbers", since the count starts at 1 and not 0.